doc/classes/TextureLayered.xml

```diff
@@ -1,10 +1,23 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="TextureLayered" inherits="Texture" version="4.0">
 	<brief_description>
-		Base class for 3D texture types.
+		Base class for texture types which represent an array of [Image]s of the same size and format.
```

Comment:
I wouldn't say it represents "an array of images". Even if internally arrays are used. The TextureLayed class is used as a parent to all composite texture types. For example, a cubemap is not an array of 6 faces, the user should think of a cubemap as a cube of 6 faces. Similarly, even though you create a Texture3D (not implemented in master yet) uses an array of images, users should think of it as a single 3-dimensional image.

Reply:
hm... how about:  
`Base class for texture types which contain the data of multiple [Image]s. Each image is of the same size and format.`